src/main/scala/intent/sbt/Runner.scala

```diff
@@ -49,12 +49,12 @@ class SbtRunner(
    * Called by SBT with all the classes that match our fingerprint.
    *
    */
-  def tasks(potentialTasks: Array[TaskDef]): Array[Task] =
+  def tasks(potentialTasks: Array[TaskDef | Null] | Null): Array[Task | Null] =
     case class Suite(td: TaskDef, structure: IntentStructure)
 
     val runner = new TestSuiteRunner(classLoader)
     var focusMode = false
-    var potentialSuites: Array[Suite] = potentialTasks
+    var potentialSuites: Array[Suite] = potentialTasks.nn.map(_.nn)
```

Comment:
**Overridden java array contents are never null**
Overridden array field of type `Array[T | Null]` should have type `Array[T]`. Use `.map(_.nn)` to fix.